### DISCUSSION TITLE
[Projects Beta] Filter bookmarks / multipled saved filters.

### DISCUSSION THREAD

**Discussion by walterra** [community/community#11293] [Projects and Issues]
As a project administrator,
I want to create multiple separate filters on a per-view basis
so that they are available as predefined filter bookmarks to define the purpose of the view.

As a project user,
I want to be able to select one of the bookmarks for initial drilldown but still be able to add more ad hoc filter criteria
so that the view still serves its original purpose, but only shows those issues / PRs that are relevant for me.

**Comment by minhna** [comment]
That's exactly what I'm looking for. With multiple filters support, views will be much more useful.
